capitolwords_ng/capitolweb/cwapi/views.py

```diff
@@ -111,3 +116,48 @@ def search_by_params(request):
         return JsonResponse(response.to_dict())
     return JsonResponse("Found nothing")
 
+@api_view(['GET'])
+def count_of_term_in_content(request, term):
+    """
+    Count the mention of a term in content
+
+    time_period(optional, default to 'month') - period of time to go
+                                                back from today
+                                                month | week | day
+
+    Example:
+        http://127.0.0.1:8000/cwapi/count/obamacare/?time_period=week
+        http://127.0.0.1:8000/cwapi/count/obamacare/
+
+    :param request:
+    :param term: term whose occurances we are counting
+    :return: number of occurances of the term in content
+    """
+    time_period = 'month'
+    if 'time_period' in request.query_params:
+        time_period = request.query_params.get('time_period')
+
+    query = get_content(term)
+    search = make_search()
+
+    # Make this 0 or just remove when we have more recent data
+    starting_delta = -365
+
+    delta = TIME_PERIODS[time_period]
+    today = datetime.datetime.today()
+    start_date = today + datetime.timedelta(starting_delta - delta)
+    end_date = today + datetime.timedelta(starting_delta)
```

Comment:
Best to be explicit here and use the `days=` keyword arg, otherwise its not clear what kind of time unit this is (unless you know that it defaults to days absent and kwargs).